engines/python/setup/djl_python/deepspeed.py

```diff
@@ -108,6 +108,7 @@ def __init__(self):
         self.model_config = None
         self.low_cpu_mem_usage = False
         self.enable_streaming = False
+        self.trust_remote_code = os.environ.get("TRUST_REMOTE_CODE", "TRUE").lower() == 'true'
```

Comment:
I think the default should be false, we don't have a way to validate that the code is safe. Users should have to explicitly opt in to downloading additional code